recipes/boost/all/conanfile.py

```diff
@@ -81,6 +82,7 @@ class BoostConan(ConanFile):
         "debug_level": 2,
         'pch': True,
         'extra_b2_flags': 'None',
+        "i18n_backend": 'iconv',
```

Comment:
hm, so recipe has no dependency on libiconv, but it still builds with iconv backend?

Reply:
yes, in Mac/Linux probably it is quite common to have it installed, `boost` finds it and uses it. If not found, and this could be a problem, `boost` won't build `locale` but it will succeed (there is a message in the output, nothing else).

Reply:
I have doubts... might be not very reliable or reproducible, then sometimes locale won't be built, but sometimes will be. maybe you can add an actual reference to libiconv recipe?

Reply:
Now I'm adding this check:

![image](https://user-images.githubusercontent.com/1406456/79993689-4fbdb080-84b5-11ea-9c49-1319509e5b97.png)

still we would need a `libiconv` requires, but probably it is better for another PR, this one keeps it as it was before (regarding `iconv`)